src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator extends Drawable {
```

Comment:
No need to make universal interfaces
```suggestion
public interface AreaCalculator {
```